Update changelog for 2.8 release (#1023)
Microsoft/dotnet-apiport: docs/VSExtension/Changelog.md

There are changes to each version of the .NET Portability Analyzer that has been released on [Visual Studio extension gallery](https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer)

## 2.8.0

__August 10, 2022__

* Server-side analysis disabled 

>     * Due to a regression in our legacy backend service, this and any future versions will work only in Offline mode

Reviewer: ```suggestion
    * We have decided to discontinue the backend service. This and all future versions will work in offline mode only.
```
